lib/chef/knife/status.rb

```diff
@@ -68,12 +72,21 @@ def run
         append_to_query("chef_environment:#{config[:environment]}") if config[:environment]
 
         if config[:hide_healthy]
+          ui.warn("-H / --hide-healthy is deprecated. Use --hide-by-mins MINS instead")
```

Comment:
Yeah, Chef::Log.deprecation is for core chef, I think we probably need to add a ui.deprecation method for knife to use, but nobody has hit that issue until now.

I wouldn't block on that.  As long as you do something where its clear enough to the user that its deprecated one way or another, then that should be good enough for now.